cl_dll/discord_integration.cpp

```diff
@@ -24,8 +25,54 @@ namespace discord_integration
 		// This seems to be consistent across PCs.
 		constexpr const char STEAM_APP_ID[] = "17215498729465839686";
 
-		// Maps for which we have thumbnails.
-		const std::unordered_set<std::string> maps_with_thumbnails {
+		// Maps in next lists must be lowercase!
+		// Discord RPC allow to upload 300 assets per app, but we also use one thumbnail as default icon, so the lists cannot contain more than 299 maps in total!
+
+		// This list was specially created for a map series, let's say that "hl1_bhop_am" series have a several versions of maps (example: hl1_bhop_am_beta1), but in fact those maps are almost identical
+		// If the beginning of map name matches with what presented in the list, then we set the same name for the thumbnail as in the list (e.g. map name: hl1_bhop_uc1_beta1, thumbnail name: hl1_bhop_uc1)
+		const std::unordered_set<std::string> map_series {
+			"hl1_bhop_am"s,
+			"hl1_bhop_bp1"s,
+			"hl1_bhop_bp2"s,
+			"hl1_bhop_faf"s,
+			"hl1_bhop_lc"s,
+			"hl1_bhop_oar"s,
+			"hl1_bhop_ocwgh"s,
+			"hl1_bhop_pu"s,
+			"hl1_bhop_qe"s,
+			"hl1_bhop_rp"s,
+			"hl1_bhop_st"s,
+			"hl1_bhop_uc1"s,
+			"hl1_bhop_uc2"s,
+		};
+
+		const std::unordered_set<std::string> dm_maps {
```

Comment:
It's better to keep these all in one map, with a comment like `// DM maps` inside. To save one map lookup.